### PR TITLE
add manifest creator-release.xml and update default.xml

### PR DESCRIPTION
**Pull request by mayank-sirotiya-imgtec**
Manifest creator-release.xml has been added for tracking releases and manifest default.xml(creator-kit.xml) has been updated for tracking master.

@CreatorKit/creators 
Signed-off-by: Mayank Sirotiya <Mayank.Sirotiya@imgtec.com>

### CODE REVIEW

**Comment by nikhil-zinjurde-imgtec** [comment]
Please update readme.md on how to use the different xmls

File: README.md [hunk 1]
```diff
@@ -35,3 +35,22 @@ After above steps, your directory structure would be like:
     ├── libobjects
     └── webscripts
 ```
+
+#### Contents
+This repository contains two different manifest files :-
+
+1. creator-kit.xml
+
+    This is the default manifest file. This manifest have repository paths which points to their master branch. Hence it is **NOT** recommended as a stable release.
```
Comment (nikhil-zinjurde-imgtec) [inline README.md]: This manifest has (instead of "have")

File: README.md [hunk 2]
```diff
@@ -35,3 +35,22 @@ After above steps, your directory structure would be like:
...
+    
+        repo init -u https://github.com/CreatorKit/manifest.git -b master
+
+2. creator-release.xml
+
+    This manifest have repository paths which points to latest stable creator kit project's release.
```
Comment (nikhil-zinjurde-imgtec) [inline README.md]: This manifest has (instead of "have")

File: creator-release.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	<project path="constrained-os/contiki" name="IMGCreator/contiki" revision="refs/tags/0.9.0"/>
+	<project path="packages/button-sensor" name="CreatorKit/button-sensor" revision="refs/tags/0.9.0"/>
+	<project path="packages/led-actuator" name="CreatorKit/led-actuator" revision="refs/tags/0.9.0"/>
+	<project path="packages/libobjects" name="CreatorKit/libobjects" revision="refs/tags/0.9.0"/>
+	<!-- Ci40 gateway repositories -->
+	<project path="dist/openwrt" name="IMGCreator/openwrt" revision="refs/tags/0.9.0"/>
+	<project path="dist/openwrt-ckt-feeds" name="CreatorKit/openwrt-ckt-feeds" revision="refs/tags/0.9.0"/>
+	<project path="dist/openwrt-feeds" name="IMGCreator/openwrt-feeds" revision="refs/tags/0.9.0"/>
+	<project path="packages/button-gateway" name="CreatorKit/button-gateway" revision="refs/tags/0.9.0"/>
+	<project path="packages/device-manager" name="CreatorKit/device-manager" revision="refs/tags/0.9.0"/>
```
Comment (nikhil-zinjurde-imgtec) [inline creator-release.xml]: Add LED controller project, as it is also there on master branch now

<project path="packages/led-controller" name="CreatorKit/led-controller" revision="refs/tags/0.9.1"/>

File: default.xml [hunk 4]
```diff
@@ -8,18 +8,18 @@
...
-	<project path="dist/openwrt-ckt-feeds" name="CreatorKit/openwrt-ckt-feeds" revision="refs/tags/0.9.0"/>
-	<project path="dist/openwrt-feeds" name="IMGCreator/openwrt-feeds" revision="refs/tags/0.9.0"/>
-	<project path="packages/button-gateway" name="CreatorKit/button-gateway" revision="refs/tags/0.9.0"/>
-	<project path="packages/device-manager" name="CreatorKit/device-manager" revision="refs/tags/0.9.0"/>
-	<project path="packages/webscripts" name="CreatorKit/webscripts" revision="refs/tags/0.9.0"/>
+	<project path="dist/openwrt" name="IMGCreator/openwrt" revision="master-pistachio"/>
+	<project path="dist/openwrt-ckt-feeds" name="CreatorKit/openwrt-ckt-feeds" revision="master"/>
+	<project path="dist/openwrt-feeds" name="IMGCreator/openwrt-feeds" revision="master"/>
+	<project path="packages/button-gateway" name="CreatorKit/button-gateway" revision="master"/>
+	<project path="packages/device-manager" name="CreatorKit/device-manager" revision="master"/>
```
Comment (nikhil-zinjurde-imgtec) [inline default.xml]: Add LED controller project, as it is also there on master branch now

<project path="packages/led-controller" name="CreatorKit/led-controller" revision="master"/>

File: creator-release.xml (hunk 3 above)
Comment (mayank-sirotiya-imgtec) [inline creator-release.xml]: There is no 0.9.1 for led-controller.. This should be added in master only as you suggested

Comment (nikhil-zinjurde-imgtec) [inline creator-release.xml]: Yes, you're right, please ignore this comment

**Comment by nikhil-zinjurde-imgtec** [comment]
Approved & verified

**Pull request merged**